### DISCUSSION TITLE
Перенос параметров в файл config zapret на linux/openwrt

### DISCUSSION THREAD

**Discussion by AcidSoap** [Storik4pro/goodbyeDPI-UI#55] [Q&A]
У меня работает 4 пресет zapret. Я попытался разобраться как перенести эти параметры в файл config, однако ничего не работает.
Я видел, что там есть кнопка показывающая с какими параметрами запускается, но мне трудно это сопоставить с файлом и куда какой параметр нужно вписывать.

**Comment by Storik4pro** [comment]
Если вы используете Linux, попробуйте запустить оригинальный блокчек на Linux. 

Пока что нету возможности автоматически конвертировать параметры запуска winws в конфигфайл для zapret на Linux
